modules/migration-launching-cam.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="migration-launching-cam_{context}"]
+= Launching the CAM web console
+
+.Procedure
+
+. If you are using self-signed certificates, launch a browser and accept the CA certificates manually for the following:
+
+* CAM host's OAuth and API server, for example, `https://_<migration-openshift-migration.apps.cluster.com>_:6443/.well-known/oauth-authorization-server`
+* Source cluster's OAuth server, for example, `https://_<master1.cluster.com>_/.well-known/oauth-authorization-server`
+* Source cluster's API server, for example, `https://_<master1.cluster.com>_/api/v1/namespaces`
+
+. Navigate to the CAM web console.
```

Comment:
Can you give users any tips on what the URL for the CAM web console is, or where to find it?

Reply:
It's mentioned twice before, in both the CORS config sections, but I guess it doesn't hurt to hit them over the head with it. Will add.